conda/recipes/pyraft/meta.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+# Usage:
+  #   conda build . -c conda-forge -c numba -c rapidsai -c pytorch
+  {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+  {% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+  {% set cuda_version='.'.join(environ.get('CUDA', 'unknown').split('.')[:2]) %}
+  {% set cuda_major=cuda_version.split('.')[0] %}
+  {% set py_version=environ.get('CONDA_PY', 36) %}
+  {% set ucx_py_version=environ.get('UCX_PY_VERSION') %}
+
+package:
+  name: pyraft
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - CC
+    - CXX
+    - VERSION_SUFFIX
+
+requirements:
+  build:
+    - python x.x
+    - setuptools
+    - cython>=0.29,<0.30
+    - cmake>=3.20.1
```

Comment:
I don't think we need cmake for the python package (yet at least)

Reply:
I see the cuml python package also has cmake in it. I'll remove it for now and add it back if needed.